pkgs/tools/audio/goxlr-utility/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, llvmPackages_11
+, pkg-config
+, libpulseaudio
+, dbus
+, clang
+, speechd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "goxlr-utility";
+  version = "v0.12.6";
+
+  src = fetchFromGitHub {
+    owner = "GoXLR-on-Linux";
+    repo = pname;
+    rev = version;
+    sha256 = "vvaKCsqncRhag8IrS0AIfNqNHGU2WIvFaYISEVfUB2Y=";
+  };
+
+  LIBCLANG_PATH = "${llvmPackages_11.libclang.lib}/lib";
```

Comment:
I think there's the `rustPlatform.bindgenHook` which could be used instead. Did you try it?

Reply:
Nope, not a rust dev myself and this solution was suggested on some thread :) Your version works and also removes the nativeBuildInput requirement for clang.